CareKit.xcodeproj/project.pbxproj

```diff
@@ -1162,6 +1167,7 @@
 		8605A5D01C4F04EC00DD65FF /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
```

Comment:
It was suggested by Xcode when updating the project to the recommended settings.

I can remove it if you think we don't need it.

If it should be removed, should it be removed both for the Debug and Release build configurations?